specs/altair/fork.md

```diff
@@ -41,6 +41,21 @@ Note that for the pure Altair networks, we don't apply `upgrade_to_altair` since
 After `process_slots` of Phase 0 finishes, if `state.slot % SLOTS_PER_EPOCH == 0` and `compute_epoch_at_slot(state.slot) == ALTAIR_FORK_EPOCH`, an irregular state change is made to upgrade to Altair.
 
 ```python
+def translate_participation(pending_attestations: Sequence[PendingAttestation], post: BeaconState) -> None:
```

Comment:
To align with other "[Beacon state mutators](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md#beacon-state-mutators)", I suggest passing:
```suggestion
def translate_participation(state: BeaconState, pending_attestations: Sequence[PendingAttestation]) -> None:
```
Reasons:
1. This function modifies the given mutable parameter.
2. There is no `pre` state here.

Reply:
Good idea, updated!